blueprints/app/files/package.json

```diff
@@ -12,17 +12,17 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "concurrently \"npm:lint:*(!fix)\" --names \"lint:\"",
+    "lint": "concurrently --group --raw \"npm:lint:*(!fix)\"",
     "lint:css": "stylelint \"**/*.css\"",
-    "lint:css:fix": "concurrently \"npm:lint:css -- --fix\"",
-    "lint:fix": "concurrently \"npm:lint:*:fix\" --names \"fix:\"",
+    "lint:css:fix": "stylelint \"**/*.css\" --fix",
+    "lint:fix": "concurrently --group --raw \"npm:lint:*:fix\"",
     "lint:hbs": "ember-template-lint .",
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache",
     "lint:js:fix": "eslint . --fix<% if (typescript) { %>",
     "lint:types": "tsc --noEmit<% } %>",
     "start": "ember serve",
-    "test": "concurrently \"npm:lint\" \"npm:test:*\" --names \"lint,test:\"",
+    "test": "concurrently --group --kill-others-on-fail --raw \"npm:lint\" \"npm:test:*\"",
```

Comment:
`--kill-others-on-fail`: make sure to exit early when the a script fails.
AFAICT, this is the closest we can get to the old behaviour when we were using `npm-run-all`.
`npm-run-all` would actually run `lint` and `test:*` sequentially. `test:*` would not be run if `lint` failed.